lib/Net/Fastly.pm

```diff
@@ -203,15 +203,34 @@ sub commands {
     return eval { $self->{__cache_commands} = $self->client->_get('/commands') };
 }
 
-=head2 purge <path>
+=head2 purge <path> [soft]
 
 Purge the specified path from your cache.
 
+You can optionally pass in a true value to enable "soft" purging e.g
+
+    $fastly->purge($url, 1);
```

Comment:
I think this should be `$service->` ?

as $fastly is the client in the rest of the docs